Makefile

```diff
@@ -1,6 +1,6 @@
 # PREAMBLE
```

Comment:
Nit: don't you want to already move Packer to 1.8 here?

Reply:
I would bump it to `1.7.9` instead to match requirements in `modules/packer/custom-image/versions.pkr.hcl`, but I would do it separately and after better version comparator is deliverd.